README.md

```diff
@@ -2,6 +2,9 @@
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/EddieHubCommunity/EventCalendar)
+
+
```

Comment:
@developer-diganta do we need the extra line on line 7?

Reply:
No I guess not. Should I remove it?

Reply:
Yes please @developer-diganta